app/components/post-search-results.js

```diff
@@ -0,0 +1,134 @@
+import Component from '@ember/component';
+import { computed } from '@ember/object';
+import { inject as service } from '@ember/service';
+import { getQueryString } from '@wikia/ember-fandom/utils/url';
+
+import config from '../config/environment';
+
+// TODO: Remove this when all discussions' posts are in the index
+const QUIZZES_WHITELIST = [
+  // test communities
+  'keikosandbox.fandom.com',
+  'keiko-test.fandom.com',
+  'gameofthrones.bartosz.fandom-dev.us',
+  'gameofthrones.chriss.fandom-dev.us',
+  'xkxd02.mattk.fandom-dev.us',
+  'xkxd02.jrogan.fandom-dev.us',
+  'xkxd02.chriss.fandom-dev.us',
+  'xkxd02.bartosz.fandom-dev.us',
+  'keikosandbox.sandbox-s3.fandom.com',
+  'keikosandbox.sandbox-s4.fandom.com',
+  'keiko-test.sandbox-s3.fandom.com',
+  'keiko-test.sandbox-s4.fandom.com',
+  // FIXME: Enable when we want to launch on Quizzes' Communities
+  // 'gameofthrones.fandom.com',
+  // 'attackontitan.fandom.com',
+  // 'marvelcinematicuniverse.fandom.com',
+  // 'marvel.fandom.com',
+  // 'southpark.fandom.com',
+  // 'starwars.fandom.com',
+  // 'strangerthings.fandom.com',
+  // 'xmenmovies.fandom.com',
+  // 'arrow.fandom.com',
+  // 'bojackhorseman.fandom.com',
+  // 'dc.fandom.com',
+  // 'dcextendeduniverse.fandom.com',
+  // 'disney.fandom.com',
+  // 'fastandfurious.fandom.com',
+  // 'godzilla.fandom.com',
+  // 'lionguard.fandom.com',
+  // 'miraculousladybug.fandom.com',
+  // 'riverdale.fandom.com',
+  // 'spongebob.fandom.com',
+  // 'tardis.fandom.com',
+  // 'thehungergames.fandom.com',
+  // '13reasonswhy.fandom.com',
+  // 'battlefield.fandom.com',
+  // 'dragonage.fandom.com',
+  // 'acecombat.fandom.com',
+  // 'borderlands.fandom.com',
+  // 'pixelgun-wiki.fandom.com',
+];
+
+
+export default Component.extend({
+  fetchService: service('fetch'),
+  logger: service(),
```

Comment:
Is this a type of dependency injection? `service()` knows what logger to use?

Reply:
Apparently. Because `logger.js` exists in `app/services`